lib/core/ogs-socknode.c

```diff
@@ -164,6 +167,8 @@ int ogs_socknode_probe(
 
         node = ogs_calloc(1, sizeof(ogs_socknode_t));
         node->addr = addr;
+        if (node->bind_dev)
```

Comment:
@pespin 

`node->bind_dev` is always NULL. ogs_socknode_probe() may not work properly.

Please review ogs_socknode_probe() for bind_to_device.

Thank you for raising this.
Sukchan

Reply:
Good catch, thanks! Fixed in next version I just pushed.